src/pages/Homepage/Homepage.tsx

```diff
@@ -143,25 +180,37 @@ const Homepage = () => {
                   dispatch(filtersActions.setLanguage(value));
                 }}
               ></Filter>
+              <Filter
+                name="Sort By"
+                options={sortByOptions}
+                onChangeValue={(value) => {
+                  dispatch(filtersActions.setSortBy(value));
```

Comment:
Interesting. Let's talk about it tomorrow, what the motivation to use redux for that.